test/test_linalg.py

```diff
@@ -5066,6 +5066,24 @@ def test_cross(self, device, dtype):
         torch.cross(x, y, out=res2)
         self.assertEqual(res1, res2)
 
+    @onlyCPU
+    @dtypes(torch.float)
+    def test_linalg_cross(self, device, dtype):
```

Comment:
Why only test on CPU? Seems like we should cover all applicable devices.

I have a few concerns with the implementation and testing:
- More test cases! Including test cases where the vector's dimension is noncontiguous, where the whole tensor is noncontiguous, where batched dimensions are noncontiguous, and where broadcast batched dimensions are noncontiguous
- To support these additional test cases I'd write a reference implementation of linalg.cross using NumPy operators and compare results with it
- For the sample inputs, I would add similar cases
- I am particularly concerned about the addition of broadcasting support in this PR torch.cross doesn't support broadcasting, right? Do you think that adding support for it will change how the gradient needs to be computed?
- If the gradient testing is especially complicated then more gradient tests (beyond sample inputs) can be added here, too

Looking forward to hearing your thoughts on this, @AnirudhDagar